Dangerfile.js

```diff
@@ -0,0 +1,39 @@
+const { customRules } = require("@pagopa/danger-plugin");
```

Comment:
This package exports more than one function?

Reply:
I made the export as default